util/types/src/core/mod.rs

```diff
@@ -27,11 +27,14 @@ mod fee_rate;
 mod reward;
 mod transaction_meta;
 mod views;
+
 pub use advanced_builders::{BlockBuilder, HeaderBuilder, TransactionBuilder};
 pub use blockchain::DepType;
 pub use extras::{BlockExt, EpochExt, EpochNumberWithFraction, TransactionInfo};
 pub use fee_rate::FeeRate;
+use once_cell::sync::OnceCell;
```

Comment:
`OnceCell` have merged into rust's `std` library: https://doc.rust-lang.org/std/cell/struct.OnceCell.html

Reply:
The once_cell in std is still a bit different from the crate we're using now, and the equivalent would be https://doc.rust-lang.org/std/sync/struct.OnceLock.html, which we can decide to replace later.